config/karma/test.karma.conf.js

```diff
@@ -1,16 +1,22 @@
 /*jshint node: true*/
 'use strict';
 
+const minimist = require('minimist');
```

Comment:
Seeing `minimist` here eats at me a little.  It would probably be possible to rewrite the test command upstream to get the karma config without relying on Karma's `parseConfig`, which expects a file path instead of a config object.  I'm just not sure it's worth all the upstream work though.

Reply:
@Blackbaud-BobbyEarl I agree that it was jarring. I'll follow your lead on what's best here!

Reply:
That's a good point.  I still feel like it could be refactored without breaking the API karma exposes.  Having said that though, I don't think these two lines warrant the amount of refactoring that would require.  I would say we proceed as written, but maybe keep it in mind if any future, more substantial changes necessitate a rewrite.